src/machines/commerce-machine.ts

```diff
@@ -133,6 +140,7 @@ export const commerceMachine = createMachine<
               },
             },
           },
+          withDefaultCoupon: {},
```

Comment:
Adds a very basic new state. There are no transitions or specific behaviors for this state, but the parent events to SWITCH or CONFIRM price will trigger any necessary transitions.

Reply:
This `withDefaultCoupon` state can also be referenced in the component if we want to conditionally render any coupon messaging. /cc @vojtaholik